### PR TITLE
backup status

### LINKED ISSUE #289: Demonstrate when backups are not finished
Once a backup starts running, indicate this on its card so, for instance, the user can immediately see a backup starts running when they first click manual snapshot when no other backups exist.

### PR DESCRIPTION
**Pull request by na3d**
Closes #289 

The next API Next Deploy will enable this feature.

### CODE REVIEW

**Comment by na3d** [comment]
screen shot 2016-08-08 at 11 11 21 am
screen shot 2016-08-08 at 11 09 05 am

**Comment by eatonphil** [comment]
Backup pending is one state and backup running is another state?

**Comment by eatonphil** [comment]
How about the actual backup thing itself in the backups page?

**Comment by eatonphil** [comment]
Another thought: pending/running backups shouldn't show up on the create page backup selection part.

**Comment by SirCmpwn** [comment]
LGTM

**Comment by na3d** [comment]
Yeah, pending and running are different states.  This may need additional tweaking after they deploy, but based on the snapshot behavior, this should work correctly.

The backup page doesn't display the status anywhere.

You mean the create page > select source > backups?

**Comment by eatonphil** [comment]
> The backup page doesn't display the status anywhere.

The primary point of #289 was to show on the backups page that there is a backup in progress but that it is not done. Currently a backup is added to the slot of backups on that page but it appears to be done when it is not.

**Comment by na3d** [comment]
A mock up for how you want this to look?

File: src/linodes/components/Backup.js [hunk 1]
```diff
@@ -14,6 +14,18 @@ export default function Backup(props) {
   const { backup, selected, onSelect } = props;
   const created = moment(backup.created);
   const cardTitle = created.calendar(null, calendar);
+  let content = created.format('dddd, MMMM D YYYY LT');
+
+  if (backup.status !== 'successful') {
+    if (backup.status === 'needsPostProcessing') {
+      content = 'Awaiting integration';
```
Comment (na3d) [inline src/linodes/components/Backup.js]: I am concerned about the wording of this one.  Opinions?

Comment (eatonphil) [inline src/linodes/components/Backup.js]: Consider them the same thing. I asked the api team to consider this as well. Users don't need to know about this second step - it's all the same backup in progress step to them.

Comment (na3d) [inline src/linodes/components/Backup.js]: Ok, so make it say running then?

Comment (eatonphil) [inline src/linodes/components/Backup.js]: That's my vote.

Comment (na3d) [inline src/linodes/components/Backup.js]: That makes sense.  I know in our current environment it is stuck in needsPostProcessing for me right now.  Are backups still broken?

**Comment by eatonphil** [comment]
Is that a hickup or do you have a backup running since yesterday?

**Comment by na3d** [comment]
The actual status is post processing.  I am guessing it is because of the other team testing things in our environment.

**Comment by eatonphil** [comment]
kk just wondering

**Comment by abemassry** [comment]
👍

**Pull request merged**